src/SideBoardCard.jsx

```diff
@@ -1,22 +1,32 @@
 import React from "react";
 import "./styles.css";
 
-export default function SideBoardCard({ value, id, total, setTotal, isVisible, setIsVisible }) {
-    // Allows sideboard cards to affect total
-    // REMOVE SIDE CARD FROM BOARD ONCE USED
-    
-    function useSideCard() {
-      if (total + value < 0) {
-        console.log("Cannot bring total below zero!");
-      } else {
-        setTotal(total + value);
-        setIsVisible(!isVisible);
-      }
+export default function SideBoardCard({ card, total, setTotal, setSideBoard }) {
+  // Allows sideboard cards to affect total
+  // REMOVE SIDE CARD FROM BOARD ONCE USED
+
+  const { id, value, isVisible } = card;
+
+  function useSideCard() {
+    if (total + value < 0) {
+      console.log("Cannot bring total below zero!");
+    } else {
+      setTotal(total + value);
+      setSideBoard((prevDeck) => {
+        const cardsDup = JSON.parse(JSON.stringify(prevDeck)); // little trick to duplicate array
```

Comment:
We can't manipulate the deck directly, we have to make a copy of it first. I use this little trick of serializing an array (turn into a string) then convert it back to an array (JSON.parse) to get a proper duplicate.